manager/src/main/scala/org/broadinstitute/transporter/transfer/TransferController.scala

```diff
@@ -49,7 +47,7 @@ class TransferController(
   /** Get the total number of transfer requests stored by Transporter. */
   def countRequests: IO[Long] =
     Fragment
-      .const(s"select count(1) from $RequestsTable")
+      .const(s"SELECT COUNT(1) from $RequestsTable")
```

Comment:
`FROM`?

Reply:
!!! crap, good catch

Reply:
Pushed the change to master: https://github.com/DataBiosphere/transporter/commit/e12afc34d7082d62d3c1309f6b45a44bf6d17ca9